README.md

```diff
@@ -32,6 +32,18 @@ or
 
 `npm run auto-changelog update --rc`
 
+#### Update the changelog for a renamed package
+
+This option is designed to be used for packages that live in a monorepo.
```

Comment:
Can this also apply to a package that is not in a monorepo, but that has just been renamed?

Reply:
@mikesposito actually I got inspired by this PR merged some time ago https://github.com/MetaMask/auto-changelog/pull/173 ? I am not sure why this won't apply for other than monorepos but my guess it should

Reply:
Oh, I see! Thanks for linking that.

As per our naming convention, release tags for packages inside a monorepo have a "tag prefix", and that tag prefix is used when building links in the changelog. As our single packages just use version as a tag, they don't need any tweak on the prefix - as they don't have it.